code/components/voip-mumble/src/MumbleAudioOutput.cpp

```diff
@@ -1331,6 +1326,24 @@ void MumbleAudioOutput::ThreadFunc()
 	CoInitialize(nullptr);
 
 	InitializeAudioDevice();
+
+	while (true) {
+		const int opusFrameTime = 40;
+
+		{
+			std::shared_lock<std::shared_mutex> _(m_clientsMutex);
+
+			for (auto& client : m_clients)
+			{
+				if (client.second)
+				{
+					client.second->PollAudio((48000 / 1000) * opusFrameTime);
+				}
+			}
+		}
+
+		std::this_thread::sleep_for(std::chrono::milliseconds(opusFrameTime));
```

Comment:
As to the above, note that when lowering sleep frequency below ~16ms care should be taken that Sleep() or wrappers for it will no longer suffice and OS scheduling APIs (waitable timers?) should be used instead.